lib/index.js

```diff
@@ -343,3 +372,13 @@ function _updateSessionAuthenticationData(
   }
   return data;
 }
+
+function _toBase64NoPad(x) {
+  return x.toString('base64', 0, 16).split('=')[0];
+}
+
+function _sleep({delay}) {
```

Comment:
As I mentioned earlier, this could use the `delay` module or the native `setTimeout` promise API: https://github.com/digitalbazaar/bedrock-authn-token-http/pull/31#discussion_r1222086558.